ipfx/bin/pipeline_from_nwb_file.py

```diff
@@ -1,41 +1,50 @@
 import allensdk.core.json_utilities as ju
-import argschema as ags
-from ipfx._schemas import GeneratePipelineInputParameters
+import sys
 import os.path
 from run_pipeline import run_pipeline
 import generate_pipeline_input as gpi
 
+OUTPUT_DIR = "/local1/ephys/patchseq/tsts"
 
 INPUT_JSON = "pipeline_input.json"
 OUTPUT_JSON = "pipeline_output.json"
 
 
```

Comment:
I can not get logging output when running from python. Is there a way to enable logging?